modules/core/03-connection/keeper/keeper_test.go

```diff
@@ -60,7 +60,7 @@ func (suite *KeeperTestSuite) TestSetAndGetClientConnectionPaths() {
 }
 
 // create 2 connections: A0 - B0, A1 - B1
-func (suite KeeperTestSuite) TestGetAllConnections() { //nolint:govet // this is a test, we are okay with copying locks
+func (suite *KeeperTestSuite) TestGetAllConnections() {
```

Comment:
might be nice to just pull these fixes as a pr to main since they are out of scope and could reduce diffs

Reply:
Happy to do that! And then rebase the feat branch for 02 router refactor when its merged, think its fine to keep em here tho